plotly/plotlyfig.m

```diff
@@ -887,6 +887,23 @@ function delete(obj)
            link_domain = strrep(plotly_domain, 'https://', ''); 
            link_domain = strrep(link_domain, 'http://', ''); 
            link_text = ['Export to ' link_domain]; 
-        end   
+        end
+
+        function sharing_value = get_sharing(obj)
+            config = loadplotlyconfig();
+            config = config.sharing;
```

Comment:
This will fail if sharing is not defined. Could you check that it exists first. If it is not defined, we should check that world_readable is defined and use that. If not we should default to `world_readable=true`.